sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseFilterForm.java

```diff
@@ -154,12 +157,18 @@ protected void applyDependenciesOnFieldChange(String propertyId, Property.ValueC
 				DistrictReferenceDto district = (DistrictReferenceDto) event.getProperty().getValue();
 
 				if (!DataHelper.equal(district, criteria.getDistrict())) {
+					getField(CaseDataDto.COMMUNITY).setValue(null);
 					getField(CaseDataDto.HEALTH_FACILITY).setValue(null);
 					getField(CaseDataDto.POINT_OF_ENTRY).setValue(null);
 				}
 
 				break;
 			}
+			case CaseDataDto.COMMUNITY:
+				CommunityReferenceDto community = (CommunityReferenceDto) event.getProperty().getValue();
+				if (!DataHelper.equal(community, criteria.getCommunity())) {
+					criteria.setHealthFacility(null);
```

Comment:
Shouldn't this also be `getField(CaseDataDto.HEALTH_FACILITY).setValue(null);`?

Reply:
fixed